packages/xo-web/src/xo-app/hub/resource.js

```diff
@@ -17,6 +17,10 @@ import { withRouter } from 'react-router'
 
 import ResourceForm from './resource-form'
 
+// For documentation about hub resources, please see www-xo documentation
```

Comment:
Is this relevant since an Open Source user can't see it?

Reply:
It was @badrAZ request, but I think it's useful for us.

Reply:
Ok then @badrAZ, what do you think?

(also, I don't get why this already changed if it was the first review round?)

Reply:
I think is not a big deal the important is to explain what the id and the namespace means but it's a plus if we have the repertory which the documentation are stored.

Reply:
So I think that it's enough to add the link of documentation